<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employeeUpdateForm</class>
 <widget class="QWidget" name="employeeUpdateForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="area" native="true">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>621</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, 
                                x1:0, y1:0, x2:0, y2:1, 
                                stop:0 rgba(6, 6, 6, 255), 
                                stop:0.41 rgba(56, 50, 52, 255), 
                                stop:1 rgba(34, 33, 33, 255));;
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Azonix</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>Update Employee :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
}

/* Hover effect */
QLineEdit:hover {
	border:3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}

/* Focus effect */
QLineEdit:focus {
    border:2px solid  qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}
</string>
    </property>
    <property name="placeholderText">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
}

/* Hover effect */
QLineEdit:hover {
	border:3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}

/* Focus effect */
QLineEdit:focus {
    border:2px solid  qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="address">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
}

/* Hover effect */
QLineEdit:hover {
	border:3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}

/* Focus effect */
QLineEdit:focus {
    border:2px solid  qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}
</string>
    </property>
    <property name="placeholderText">
     <string>address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneNum">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
}

/* Hover effect */
QLineEdit:hover {
	border:3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}

/* Focus effect */
QLineEdit:focus {
    border:2px solid  qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Phone Number</string>
    </property>
   </widget>
   <widget class="QComboBox" name="department">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
    border: 2px solid transparent; /* Default border */
}

/* Drop-down arrow area */
QComboBox::drop-down {
    border: none;
    width: 30px;
   ; /* Keep background for drop-down */
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
    image: url(:/textures/textures/downArrowIcon.png);
}

/* Hover effect */
QComboBox:hover {
    border: 3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));
}

/* Focus effect */
QComboBox:focus {
    border: 2px solid qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));
}

/* Style for the dropdown list */
QComboBox QAbstractItemView {
    background-color: white;
    border: 2px solid #C51D34;
    selection-background-color: #ff0004;
    selection-color: white;
    border-radius: 8px;
    color: black;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Update Department</string>
    </property>
    <item>
     <property name="text">
      <string>Human Resources</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Federation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Municipality</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="face">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
}

/* Hover effect */
QLineEdit:hover {
	border:3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}

/* Focus effect */
QLineEdit:focus {
    border:2px solid  qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}
</string>
    </property>
    <property name="placeholderText">
     <string>face id </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="voice">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
}

/* Hover effect */
QLineEdit:hover {
	border:3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}

/* Focus effect */
QLineEdit:focus {
    border:2px solid  qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}
</string>
    </property>
    <property name="placeholderText">
     <string>voice id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_exit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  
border-radius:10px;
color:white;
}
QPushButton:hover {
    background-color:qlineargradient(x1: 0.25, y1: 0, x2: 0.75, y2: 1, 
                            stop: 0 rgba(0, 255, 254, 255),     
                            stop: 0.26 rgba(38, 170, 196, 255), 
                            stop: 0.67 rgba(89, 85, 131, 255),
                            stop: 1 rgba(0, 238, 255, 255));  
}</string>
    </property>
    <property name="text">
     <string>Save and Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  
border-radius:10px;
color:white;
}
QPushButton:hover {
    background-color:qlineargradient(x1: 0.25, y1: 0, x2: 0.75, y2: 1, 
                            stop: 0 rgba(0, 255, 254, 255),     
                            stop: 0.26 rgba(38, 170, 196, 255), 
                            stop: 0.67 rgba(89, 85, 131, 255),
                            stop: 1 rgba(0, 238, 255, 255));  
}</string>
    </property>
    <property name="text">
     <string>Cancel Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="userIDLabel">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>12</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Azonix</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:yellow;
</string>
    </property>
    <property name="text">
     <string>label</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="position">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
}

/* Hover effect */
QLineEdit:hover {
	border:3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}

/* Focus effect */
QLineEdit:focus {
    border:2px solid  qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}
</string>
    </property>
    <property name="placeholderText">
     <string>position</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
